apps/passport-server/src/services/devconnect/organizerSync.ts

```diff
@@ -762,15 +837,20 @@ export class OrganizerSync {
             );
           }
           const email = (attendee_email || order.email).toLowerCase();
+          const checkin_timestamp =
+            checkins.length > 0 ? new Date(checkins[0].datetime) : null;
 
           tickets.push({
             email,
             full_name: attendee_name,
             devconnect_pretix_items_info_id: existingItem.id,
             is_deleted: false,
-            is_consumed: false,
+            is_consumed: checkin_timestamp !== null,
```

Comment:
```suggestion
            is_consumed: checkin_timestamp != null,
```

this way it covers both `null` and `undefined`

Reply:
Why would it be undefined? Unless node-postgres translates null database values to undefined...

Reply:
To be clear, the variable is set above:
`          const pretix_checkin_timestamp =
            checkins.length > 0 ? checkins[0].datetime : null;`

It does not seem to be possible that the result is undefined. `checkins[0]` exists, and `checkins[0].datetime` is typed as `Date | null`.